packages/p2p/src/stores/my-ads-store.js

```diff
@@ -363,7 +363,7 @@ export default class MyAdsStore extends BaseStore {
             ],
             offer_amount: [
                 v => !!v,
-                v => (values.type === buy_sell.SELL ? v <= this.available_balance : !!v),
+                v => (values.type === buy_sell.SELL && !isNaN(values.offer_amount) ? v <= this.available_balance : !!v),
```

Comment:
And this is not necessary, `!isNaN` already exists on the next line, this error is just being prioritized over that one. So all you have to do is move line 367 above this one and similarly update the errors.

Reply:
Yes, it's exists, but doesn't work for Offer_amount value with 'sell USD' option, it's showing the error 'Max available amount is 10000' even when we enter invalid input @carol-binary